Testscripts/Linux/RELOAD-MODULES.sh

```diff
@@ -118,9 +118,17 @@ VerifyModules()
 # Main script body
 #
 #######################################################################
+if (printf '%s\n' "${HYPERV_MODULES[@]}" | grep -xq "hv_utils"); then
+    if modprobe -r hv_utils; then
+        msg="Error: hv_utils could be disabled!"
+        LogMsg "${msg}"
+        SetTestStateFailed
+        exit 0
+    fi
+fi
+
 if (printf '%s\n' "${HYPERV_MODULES[@]}" | grep -xq "hyperv_fb"); then
-    modprobe -r hyperv_fb
-    if [[ $? -eq 0 ]]; then
+    if modprobe -r hyperv_fb; then
```

Comment:
> if modprobe -r hv_utils; then
> echo yes
> fi
modprobe: FATAL: Module hv_utils is in use.

seems it won't enter into if block when throw in use error

Reply:
Thanks Lili!
Code updated.